src/overlays/actors/ovl_Demo_Syoten/z_demo_syoten.h

```diff
@@ -7,9 +7,41 @@ struct DemoSyoten;
 
 typedef void (*DemoSyotenActionFunc)(struct DemoSyoten*, GlobalContext*);
 
+#define DEMOSYOTEN_GET_F(thisx) ((thisx)->params & 0xF)
+#define DEMOSYOTEN_GET_7E00(thisx) (((thisx)->params & 0x7E00) >> 9)
+
+#define DEMOSYOTEN_7E00_3F 0x3F
+
+enum {
+    /* 0 */ DEMOSYOTEN_F_0,
+    /* 1 */ DEMOSYOTEN_F_1,
+    /* 2 */ DEMOSYOTEN_F_2,
+    /* 3 */ DEMOSYOTEN_F_3,
+    /* 4 */ DEMOSYOTEN_F_4
+};
+
+typedef struct {
+    /* 0x00 */ UNK_TYPE1 unk00[0x1];
+    /* 0x01 */ u8 count;
+    /* 0x02 */ UNK_TYPE1 unk2[0x16];
+} DemoSyotenStruct2; // almost certainly wrong, just filling space
+
 typedef struct DemoSyoten {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x2B0];
+    /* 0x0144 */ UNK_TYPE1 unk_144[0x18];
+    /* 0x015C */ DemoSyotenStruct2* unk_15C;
+    /* 0x0160 */ UNK_TYPE1 unk160[0x14];
```

Comment:
This is probably all one struct, or at least a struct with substructs, given that almost none of the rest of it is referenced. But it's pretty meaningless without c_keyframe anyway.